src/components/DelegationDashboard/Chat/NewChat.tsx

```
@@ -0,0 +1,175 @@
+			setLoading(false);
+		} else if (error) {
+			console.log(error);
+			setLoading(false);
+		}
+	};
+
+	const handleSearch = (searchTerm: string) => {
+		setSearchAddress(searchTerm);
+		if (searchTerm) {
+			const filteredDelegates = allDelegates.filter(
+				(delegate) => delegate.address.toLowerCase().includes(searchTerm.toLowerCase()) || (delegate.username && delegate.username.toLowerCase().includes(searchTerm.toLowerCase()))
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)